sdk/servicebus/Azure.Messaging.ServiceBus/src/ServiceBusMessage.cs

```diff
@@ -16,22 +16,12 @@ namespace Azure.Messaging.ServiceBus
     /// </remarks>
     public class ServiceBusMessage
```

Comment:
Some basic properties are missing. Issue https://github.com/Azure/azure-sdk-for-net/issues/9886 was closed w/o mentioning any continuation of this work. I find it hard to believe that there will be no further work on the message models 🙂

Reply:
I reopened the issue so that we can discuss there. I believe the only deltas from your comments are the ForcePersistence and State properties which I didn't see in the Track 1 version either.